independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientBuilderImpl.java

```diff
@@ -238,6 +250,16 @@ public ClientImpl build() {
 
     }
 
+    private void configureNonProxyHosts(HttpClientOptions options, String nonProxyHosts) {
+        if (nonProxyHosts != null) {
+            for (String host : nonProxyHosts.split(PIPE)) {
+                if (!host.isBlank()) {
+                    options.addNonProxyHost(host);
```

Comment:
Does it support the format for the JVM nonProxyHosts (e.g. wildcards)?

Reply:
yes

Reply:
Cool, thanks for confirming.

BTW, I insisted on it because it's very common in enterprise setups to have a proxy for external websites and to have to avoid it for internal sites. And that could happen when using the REST client.

Reply:
here's an example from vertx pr that introduced it: https://github.com/eclipse-vertx/vert.x/commit/1e95256b84ff9684d53f4c8383008f49613cd5e0#diff-4d9e05c07eed6e23cc3e1e3e32861b7689d8334e963d6bedb7531c00c3853c6aR1031

Reply:
> Cool, thanks for confirming.
> 
> BTW, I insisted on it because it's very common in enterprise setups to have a proxy for external websites and to have to avoid it for internal sites. And that could happen when using the REST client.

sure. I missed it in the original PR, it wasn't omitted intentionally